apps/vaults/hooks/useSolverCowswap.ts

```diff
@@ -245,13 +266,13 @@ export function useSolverCowswap(): TSolverContext {
 				const {isSuccessful, error} = await checkOrderStatus(orderUID, quote.validTo);
 				if (error) {
 					console.error(error);
-					return ({isSuccessful: false, error});
+					return ({isSuccessful: false, error: error as BaseError || ''});
```

Comment:
This empty string doesn't feel right, also worth to send these errors to Sentry.